lib/rex/parser/fs/ntfs.rb

```diff
@@ -213,10 +217,11 @@ def mft_record_attribute(mft_record)
             res[attribute_identifier] = mft_record[curs + content_offset, content_size]
           else
             # non resident
-            if attribute_identifier == DATA_ATTRIBUTE_ID
-              res[attribute_identifier] = mft_record[curs, attribute_size]
-            else
+            if attribute_identifier == INDEX_ALLOCATION_ID or 
+              (not lazy and attribute_identifier != DATA_ATTRIBUTE_ID)
```

Comment:
@zeroSteiner the attribute DATA, must not be gathered even in not lazy mode (too huge).
the attribute index_allocation must always be gathered, it's necessary to parse directory.
Otherway, the attribute must be gathered only in not lazy mode.
I think that's what i implement.